.github/workflows/tests.yml

```diff
@@ -193,10 +193,12 @@ jobs:
     runs-on: ${{ matrix.os }}
     services:
       elasticsearch:
-        image: elasticsearch:7.17.6
+        image: elasticsearch:8.6.0
         env:
           discovery.type: "single-node"
+          xpack.security.enabled: "false"
           ES_JAVA_OPTS: "-Xms128m -Xmx256m"
+          ELASTIC_CLIENT_APIVERSIONING: 1
```

Comment:
I think setting it `1` works but the [official docs](https://www.elastic.co/guide/en/elasticsearch/reference/8.6/rest-api-compatibility.html#request-rest-api-compatibility) say to set it to `true`. 🤷‍♀️